doc/api/events.md

```diff
@@ -155,9 +155,63 @@ myEmitter.emit('error', new Error('whoops!'));
 // Prints: whoops! there was an error
 ```
 
+## AutoCatch of Promise rejection
+
+Using async functions in event handlers is problematic, because it would
+lead to `'unhandledRejection'` in case of an exception
+
+```js
+const ee = new EventEmitter();
+ee.on('something', async (value) => {
+  throw new Error('kaboom');
+});
+```
+
+The `autoCatch` option in the `EventEmitter` constructor or the global
```

Comment:
Please add a test on how this works in subclasses of `EventEmitter`? This is scary for libraries to do otherwise I think.